data/AryanParashar24.json

```diff
@@ -0,0 +1,38 @@
+{
+"name": "Aryan Parashar",
+  "type": "personal",
+  "bio": " 19 🧑‍💻 | DevOps 🥑 ,Open Source💜 & Web development enthusiast | Leaning  ML, Cloud native computing☁️ & MERN- Stack Development 🌐",
+  "avtar": "https://github.com/AryanParashar24.png" ,
```

Comment:
Remove this line. Currently, you don't need this line.  

```suggestion
```

Reply:
Make sure you commit changes to [AryanParashar24-patch-2-3](https://github.com/AryanParashar24/LinkFree/tree/AryanParashar24-patch-2-3), to avoid making any other duplicate PR further for the changes requested above.